pkgs/by-name/go/godot_4/package.nix

```diff
@@ -34,13 +37,13 @@
   withTarget ? "editor",
   withTouch ? true,
   withUdev ? true,
+  withWayland ? true,
+  withX11 ? true, # compilation without X11 fails until this fix is merged: https://github.com/godotengine/godot/pull/73504
```

Comment:
The comment should precede the 'withWayland' since this is a problem affecting Wayland. (X11 is fine.)

It should be two lines because of limit of characters. Like:

```
> # Wayland support in Godot requires X11 dependencies until upstream fix is merged
> https://...
> withWayland ? true
```